### ISSUE TITLE
magento-scripts cleanup: No config found for magento version undefined

### ISSUE THREAD

**Issue by Morgy93** [scandipwa/create-magento-app#60]
### Describe the bug

The `npm run cleanup` fails with error `No config found for magento version undefined`
```
> create-magento@0.0.0 cleanup
> magento-scripts cleanup

WARNING! you should not use this command.
❯ Cleanup project
  ✔ Checking configuration file
  ✖ No config found for magento version undefined
  ◼ Stopping PHP-FPM
  ◼ Removing volumes
  ◼ Cleaning cache
  ◼ Uninstall Magento App
  ◼ Remove magento application folder
magento-scripts cleanup

Cleanup project folder

Options:
      --help     Show help                                             [boolean]
      --version  Show version number                                   [boolean]
  -f, --force    Force cleanup to remove app directory[boolean] [default: false]

Error: No config found for magento version undefined
    at getConfigFromMagentoVersion (create-magento-app/node_modules/@scandipwa/magento-scripts/lib/config/index.js:41:19)
    at Task.task (create-magento-app/node_modules/@scandipwa/magento-scripts/lib/config/get-project-configuration.js:11:28)
    at Task.run (create-magento-app/node_modules/listr2/dist/lib/task.js:213:45)
```

### Did you try recovering your dependencies?

Why? I'm trying to delete a newly installed app.

### Environment

WSL2 Ubuntu

### Steps to reproduce

1. Install new app
2. Add `"cleanup": "magento-scripts cleanup"` to package.json
3. Run `npm run cleanup`

### Expected behavior

The app removes itself

### Actual behavior

An error is thrown

**Comment by ejnshtein** [comment]
Hi!

Thanks for the report!

Yeah, `cleanup` command wasn't tested for some time, I will address this issue in the next patch update.

**Comment by ejnshtein** [comment]
The issue is fixed in version [1.13.3](https://github.com/scandipwa/create-magento-app/releases/tag/%40scandipwa%2Fmagento-scripts%401.13.3).
